src/Stan/Observation.hs

```diff
@@ -24,7 +24,8 @@ import Stan.Inspection (Inspection)
 data Observation = Observation
     { observationId           :: !(Id Observation)
     , observationInspectionId :: !(Id Inspection)
-    , observationLoc          :: !SrcSpan
+    , observationLoc          :: !RealSrcSpan
+    , observationFile         :: !FilePath
```

Comment:
I went for the simplest solution for now and just added `FilePath` as a field. We can change this later...